src/e2e-test/java/io/cdap/plugin/stepsdesign/BigQueryCommonSteps.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.stepsdesign;
+
+import io.cdap.e2e.pages.actions.CdfBigQueryPropertiesActions;
+import io.cdap.e2e.pages.actions.CdfPipelineRunAction;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.plugin.tests.hooks.TestSetupHooks;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.Assert;
+
+import java.io.IOException;
+
+/**
+ * Represents common steps of BigQuery plugins.
+ */
+public class BigQueryCommonSteps {
```

Comment:
This class can be removed. Lets discuss in call.

Reply:
Removed the class. It just I needed the variables (Target Dataset and Table) public and static in hooks because they are being used in validation logic which is in Actions Class.

Reply:
Variables can also be removed.
You can access using `PluginPropertyUtils` wherever required.
like `PluginPropertyUtils.pluginProp("bqtarget.table") `

Reply:
Removed the global variables create local ones and accessed using  PluginPropertyUtils (b0aea7b896bb3d127a9a5952d501e846a4c3efda)